scripts/dts/new_extract.py

```diff
@@ -55,26 +55,39 @@ def main():
             print("#define DT_SRAM_BASE_ADDRESS\t"
                       + hex(edt.sram_dev.regs[0].addr),
                   file=out)
+            print("#define DT_SRAM_SIZE\t"
+                      + ascii(int(edt.sram_dev.regs[0].size / 1024)),
+                  file=out)
 
         if edt.ccm_dev:
             print("#define DT_CCM_BASE_ADDRESS\t"
                       + hex(edt.ccm_dev.regs[0].addr),
                   file=out)
+            print("#define DT_CCM_SIZE\t"
+                      + ascii(int(edt.ccm_dev.regs[0].size / 1024)),
+                  file=out)
 
 
 def write_regs(dev, out):
     for reg in dev.regs:
-        print("#define {}\t0x{:x}".format(reg_ident(reg), reg.addr), file=out)
+        print("#define {}\t0x{:x}".format(reg_ident(reg, "BASE_ADDRESS"), reg.addr), file=out)
```

Comment:
Could add `BASE_ADDRESS` to the `"#define ..."` string instead of passing it as an argument to `reg_ident()`.

Reply:
Hmm... it's not always added to the end though.

Maybe it's fine. Will need to fix the documentation `reg_ident()` to mention what the `prop` argument does though.